mypy.ini

```diff
@@ -70,29 +70,7 @@ no_implicit_optional = true
 warn_return_any = true
 warn_unreachable = true
 
-[mypy-pytradfri.api.__init__]
-check_untyped_defs = true
-disallow_incomplete_defs = true
-disallow_subclassing_any = true
-disallow_untyped_calls = true
-disallow_untyped_decorators = true
-disallow_untyped_defs = true
-no_implicit_optional = true
-warn_return_any = true
-warn_unreachable = true
-
-[mypy-pytradfri.api.aiocoap_api]
-check_untyped_defs = true
-disallow_incomplete_defs = true
-disallow_subclassing_any = true
-disallow_untyped_calls = true
-disallow_untyped_decorators = true
-disallow_untyped_defs = true
-no_implicit_optional = true
-warn_return_any = true
-warn_unreachable = true
-
-[mypy-pytradfri.device.controller]
+[mypy-pytradfri.api]
```

Comment:
I think we need a star pattern to glob all submodules in a directory. The libcoap api isn't typed so we're not globbing correctly here, since the build passes.

Reply:
You're right, and when adding * I also noticed we haven't typed `based_controller` properly. I'll close this PR and open again when we're through with that.